Candidate: CVE-2023-25399
PublicDate: 2023-07-06
References:
 https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2023-25399
 https://github.com/scipy/scipy/issues/16235
 https://github.com/scipy/scipy/pull/16397
 http://www.square16.org/achievement/cve-2023-25399/
Description:
 A refcounting issue which leads to potential memory leak was discovered in
 scipy commit 8627df31ab in Py_FindObjects() function.
Ubuntu-Description:
Notes:
Mitigation:
Bugs:
Priority: medium
Discovered-by:
Assigned-to: 0xnishit
CVSS:

Patches_scipy:
upstream_scipy: needs-triage
trusty_scipy: ignored (end of standard support)
xenial_scipy: ignored (end of standard support)
bionic_scipy: ignored (end of standard support)
focal_scipy: needs-triage
esm-apps/focal_scipy: needs-triage
jammy_scipy: needs-triage
esm-apps/jammy_scipy: needs-triage
kinetic_scipy: needs-triage
lunar_scipy: needs-triage
devel_scipy: needs-triage
